torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -357,37 +400,40 @@ void ExportModule(
     const Module& module,
     std::ostream& out,
     const ExtraFilesMap& extra_files,
-    bool bytecode_format) {
+    bool bytecode_format,
+    bool save_debug_info) {
```

Comment:
Actually given that we are saving this only for byte code right now, lets call it save_debug_info_in_bytecode.